### DISCUSSION TITLE
Number of votes cast is incorrect?

### DISCUSSION THREAD

**Discussion by denvercitizen9** [snapshot-labs/snapshot#2845] [Q&A]
The voter was able to cast a vote, but it appears that their **votes only counted as 1 vote, despite them having 2 voting tokens.**  The only place I can see how many votes they cast is to click "see more" under votes to see the voter (webthethird.eth)

They should have had 2 votes, not 1 vote.  I'm not even sure where to see the details related to the snapshot.  How to figure out what went wrong?

**Proposal**
[CityDAO CIP 69](https://snapshot.org/#/daocity.eth/proposal/0x2b54d9b32411103a6d55d9efe74500a60b123d9250d1f3fd72d32f60ed19a657)

**Voter Address**
[etherscan: 0x830c91140AD5851D5b765691e14c04101829Eaea](https://etherscan.io/address/0x830c91140AD5851D5b765691e14c04101829Eaea#tokentxnsErc1155)
Note: items in question are the two most recent 1155 tokens w/ID of 42

**Receipt**
[IPFS](https://snapshot.mypinata.cloud/ipfs/bafkreiam4irx7zxcgkiluw34uzmeifkwmi6dzvbpeoaq6scpn3fkgwrwd4)

**Comment by denvercitizen9** [comment]
It appears to be this strategy, but having a hard time understanding where the numbers come from (and a link to see the actual strategy would be good also):
https://snapshot.org/#/playground/citydao-square-root?query=eyJwYXJhbXMiOnsic3ltYm9sIjoiQ0lUSVpFTiIsImFkZHJlc3MiOiIweDdFZUY1OTFBNkNDMDQwM2I5NjUyRTk4RTg4NDc2ZmUxYkYzMWREZWIiLCJ0b2tlbklkIjo0MiwiZGVjaW1hbHMiOjB9LCJuZXR3b3JrIjoiMSIsInNuYXBzaG90IjoiMTUxNjMzMDMiLCJhZGRyZXNzZXMiOltdfQ..

**Reply by samwhosung** [reply]
You can find the link by going a step back and then clicking on the version number. Here is a link https://github.com/snapshot-labs/snapshot-strategies/tree/master/src/strategies/citydao-square-root

**Comment by samwhosung** [comment]
@ChaituVR any ideas why the user is missing 1 voting power?
